include/psa/crypto_values.h

```diff
@@ -1538,9 +1538,8 @@
  *         algorithm identifier.
  */
 #define PSA_ALG_IS_HASH_AND_SIGN(alg)                                   \
-    (PSA_ALG_IS_RSA_PSS(alg) || PSA_ALG_IS_RSA_PKCS1V15_SIGN(alg) ||    \
-     PSA_ALG_IS_ECDSA(alg) || PSA_ALG_IS_HASH_EDDSA(alg) ||             \
-     PSA_ALG_IS_VENDOR_HASH_AND_SIGN(alg))
+    (PSA_ALG_IS_RSA_PSS(alg) || PSA_ALG_IS_ECDSA(alg) ||                \
```

Comment:
Missing: RSA_PKCS1V15_SIGN other than raw. There isn't a macro for that, you can use
```
PSA_ALG_IS_RSA_PKCS1V15_SIGN(alg) && (alg) != PSA_ALG_RSA_PKCS1V15_SIGN_RAW
```